components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/CacheBackedIdPMgtDAO.java

```diff
@@ -517,6 +517,50 @@ public IdentityProvider getIdPByRealmId(String realmId, int tenantId,
         return identityProvider;
     }
 
+    /**
+     * Get the enabled IDP of the given realm id
+     *
+     * @param realmId
```

Comment:
Let's add param descriptions and return descriptions. Check similar places

Reply:
Fixed with [87fcdcc](https://github.com/wso2/carbon-identity-framework/pull/4991/commits/87fcdcccab97dd5619bded3e551b7410a7520772)